kinto/core/resource/schema.py

```diff
@@ -400,27 +400,27 @@ class NotModifiedResponseSchema(colander.MappingSchema):
     header = ResponseHeaderSchema()
 
 
-class RecordResponseSchema(colander.MappingSchema):
+class ObjectResponseSchema(colander.MappingSchema):
     """Response schema used with sigle resource endpoints."""
 
     header = ResponseHeaderSchema()
 
     @colander.deferred
     def body(node, kwargs):
-        return kwargs.get("record")
+        return kwargs.get("object")
```

Comment:
I'm not sure I really understand what these `kwargs` are and whether I should expect them to have an `object` or a `record`.